### ISSUE TITLE
Write main application loop to build a hand of cards

### ISSUE THREAD

**Issue by nibsi** [CoderanchCorral/Blackjack#25]
- Add main method to `Hand`.
- Repeatedly ask user to "hit" or "pass".
- If the user hits, another card must be added to their hand. The new hand is displayed.
- If the hand goes bust after hitting, a message is displayed to the user and the application terminates.
- If the user passes, the application terminates.

Depends on #21
